packages/utils/src/helpers/index.ts

```diff
@@ -160,3 +164,19 @@ export const Result = {
     return { ok: false as const, error };
   },
 };
+
+/**
+ * Return a properly formatted BIP-044 path
+ *
+ * @param {number} coinType - SLIP-044 Coin designation
+ * @param {Bip44Path} bip44Path - path object
+ * @returns {string}
+ */
+export const makeBip44Path = (
+  coinType: number,
+  bip44Path: Bip44Path
+): string => {
+  const { account, change, index } = bip44Path;
```

Comment:
`index` is optional on `Bip44Path` so if it is undefined this will return something like `"m/44'/877'/0'/0/undefined"`.

Reply:
Good catch! Also, once fixed, I should probably update the other instances where it can be utilized.

Reply:
I pushed up a fix, and added it to Keyring where an address _without_ an index needs to be formatted (mnemonic account)